e2e/testdata/hydration/namespace-repo/base/kustomization.yaml

```diff
@@ -17,5 +17,6 @@ resources:
 - role.yaml
 - networkpolicy.yaml
 
-commonLabels:
-  test-case: hydration
\ No newline at end of file
+labels:
+- pairs:
+    test-case: hydration
```

Comment:
unless it breaks something, all files should end with a training linebreak.

Reply:
added